orchagent/qosorch.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
as long as other places are using #ifndef just do the similar thing here to meet code unification